tracee/event_monitor_ebpf.c

```diff
@@ -394,6 +394,22 @@ enum event_id {
     SYS_STATX,
     SYS_IO_PGETEVENTS,
     SYS_RSEQ,
+    RESERVED335,
+    RESERVED336,
+    RESERVED337,
+    RESERVED338,
+    RESERVED339,
+    RESERVED340,
+    RESERVED341,
+    RESERVED342,
+    RESERVED343,
+    RESERVED344,
+    RESERVED345,
+    RESERVED346,
+    RESERVED347,
+    RESERVED348,
+    RESERVED349,
+    RAW_SYSCALLS,
```

Comment:
Maybe you want to add RAW_SYSCALLS to the next commit?

Reply:
Yes it’s slipped here. Will update